platform/openide.actions/src/org/openide/actions/HeapView.java

```diff
@@ -47,301 +42,104 @@
-            c = new Color(0xA6A295);
-        border1Color = c;
-
-        c = UIManager.getColor( "nb.heapview.border2" ); //NOI18N
-        if( null == c )
-            c = new Color(0xC0BCAD);
-        border2Color = c;
-
-        c = UIManager.getColor( "nb.heapview.border3" ); //NOI18N
-        if( null == c )
-            c = Color.WHITE;
-        border3Color = c;
-
-        c = UIManager.getColor( "nb.heapview.mintick.color" ); //NOI18N
-        if( null == c )
-            c = new Color(0xC7D6AD);
-        minTickColor = c;
-
-        c = UIManager.getColor( "nb.heapview.maxtick.color" ); //NOI18N
-        if( null == c )
+        Color c = UIManager.getColor("nb.heapview.maxtick.color"); //NOI18N
+        if (null == c) {
             c = new Color(0x615d0f);
-        maxTickColor = c;
+        }
+        CHART_COLOR = c;
 
-        c = UIManager.getColor( "nb.heapview.textblur" ); //NOI18N
-        if( null == c )
+        c = UIManager.getColor("nb.heapview.foreground"); //NOI18N
+        if (null == c) {
             c = Color.WHITE;
-        textBlurColor = c;
+        }
+        TEXT_COLOR = c;
 
-        c = UIManager.getColor( "nb.heapview.foreground" ); //NOI18N
-        if( null == c )
-            c = Color.WHITE;
-        textColor = c;
-
-        c = UIManager.getColor( "nb.heapview.background1" ); //NOI18N
-        if( null == c )
-            c = new Color(0xD0CCBC);
-        background1Color = c;
-
-        c = UIManager.getColor( "nb.heapview.background2" ); //NOI18N
-        if( null == c )
-            c = new Color(0xEAE7D7);
-        background2Color = c;
-
-        c = UIManager.getColor( "nb.heapview.grid1.start" );
-        if( null == c )
-            c = new Color(0xE3DFCF);
-        GRID_COLORS[0] = c;
-
-        c = UIManager.getColor( "nb.heapview.grid1.end" );
-        if( null == c )
-            c = new Color(0xE7E4D3);
-        GRID_COLORS[1] = c;
-
-        c = UIManager.getColor( "nb.heapview.grid2.start" );
-        if( null == c )
-            c = new Color(0xDAD7C6);
-        GRID_COLORS[2] = c;
-
-        c = UIManager.getColor( "nb.heapview.grid2.end" );
-        if( null == c )
-            c = new Color(0xDFDCCB);
-        GRID_COLORS[3] = c;
-
-        c = UIManager.getColor( "nb.heapview.grid3.start" );
-        if( null == c )
-            c = new Color(0xD3CFBF);
-        GRID_COLORS[4] = c;
-
-        c = UIManager.getColor( "nb.heapview.grid3.end" );
-        if( null == c )
-            c = new Color(0xD7D3C3);
-        GRID_COLORS[5] = c;
-
-        c = UIManager.getColor( "nb.heapview.grid4.start" );
-        if( null == c )
-            c = new Color(0xCECABA);
-        GRID_COLORS[6] = c;
-
-        c = UIManager.getColor( "nb.heapview.grid4.end" );
-        if( null == c )
-            c = new Color(0xD0CCBC);
-        GRID_COLORS[7] = c;
+        c = UIManager.getColor("nb.heapview.background1"); //NOI18N
+        if (null == c) {
+            c = new Color(0xC0BCAC);
```

Comment:
For this one, use 0xCEDBE6 instead. That will go well with the other suggested color change. (It's the light blueish shadow in the NetBeans logo cube.)

Perhaps also rename this one to nb.heapview.background (without the 1 at the end). This will nullify any LAF customizations from the past, but with the updated branding-consistent colors, that should be fine. New tweaks can be made e.g. for FlatLAF based on the blueish on-brand colors.